lib/access-context.js

```diff
@@ -309,6 +333,28 @@ AccessRequest.prototype.exactlyMatches = function(acl) {
 };
 
 /**
+ * Settle the accessRequest's permission if DEFAULT
+ * In most situations, the default permission can be resolved from the nested model
+ * config. An default permission can also be explicitly provided to override it or
+ * cope with AccessRequest instances without a nested model (e.g. model is '*')
+ *
+ * @param {String} defaultPermission (optional) the default permission to apply
+ */
+
+AccessRequest.prototype.settleDefaultPermission = function(defaultPermission) {
+  if (this.permission !== 'DEFAULT')
+    return;
+
+  var modelName = this.model;
+  if (!defaultPermission) {
+    var modelClass = this.registry.findModel(modelName);
+    defaultPermission = modelClass && modelClass.settings.defaultPermission;
+  }
+
+  this.permission = defaultPermission || 'ALLOW';
+};
+
+/**
  * Is the request for access allowed?
```

Comment:
new accessrequest instance method to help keeping the code tidier.
see code jsdoc comment